etsd/keys/templates/keys/publickey_detail.html

```diff
@@ -44,6 +72,9 @@ <h5>Key text</h5>
 
 {% block extra_script %}
 <script src='{% static "openpgp/openpgp.min.js" %}'></script>
+<script src='{% static "jquery-3.5.1.min.js" %}'></script>
```

Comment:
The jquery/popper/bootstrap scripts are included in the theme base.html and should not be re-used here.

Reply:
Using theme_base's bootstrap doesn't display the modal for some reason. Trying to figure out why.

Reply:
Have you tried running something like: `bootstrap5Alert({message: 'koko', title: 'toto'})` ? Please take a look at the `confirmDelete` function of the bootstrapaler.js file